.github/ISSUE_TEMPLATE/new-task-template.md

```diff
@@ -0,0 +1,18 @@
+---
+name: New Task Template
+about: Use this template when you are creating a new task
+title: ''
+labels: 'Type: Feature'
```

Comment:
Do tasks always imply features?

Reply:
Yeah, doesn't quite make sense. I replaced this with 3 templates instead: Feature, Bug, Maintenance